__tests__/actions/api.js

```diff
@@ -2,7 +2,12 @@
 
 import nock from 'nock'
 
-import {routingQuery} from '../../lib/actions/api'
+import * as api from '../../lib/actions/api'
+
+// Use mocked randId function and pass in searchId for routingQuery calls so that
+// snapshots are deterministic (i.e., the random IDs don't change).
+let idCounter = 1234
+const randId = () => `abcd${idCounter++}`
```

Comment:
I'm not a huge fan of this since it's injecting a test function into the code. Perhaps we should just consider introducing a uuid dependency so that we can do something like this: https://github.com/conveyal/analysis-ui/blob/dev/__mocks__/uuid.js